UnoCpp/UnoCpp/Card.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+#include "ColorType.h"
+
+class Card
+{
+public:
+    virtual ~Card() = default;
```

Comment:
Since the destructor is explicitly declared here, I would also include the constructor.

Reply:
I only declared it here because of the inheritance problem when you try to delete a Child class casted as its Parent.
But either way, this is just a initial version, I'll probably have to add constructors in the next commits.